module/input/Camera/src/Camera.cpp

```diff
@@ -299,9 +299,9 @@ namespace input {
                                       })));
 
             // Get torso to head, and torso to world
-            Eigen::Affine3d Htp(sensors.Htp[ServoID::HEAD_PITCH]);
+            Eigen::Affine3d Htx(sensors.Htx[ServoID::HEAD_PITCH]);
```

Comment:
This shouldn't be Htx here, this should be something real like Hth (for head), or it looks like before it was Htp (which is why it was all replaced here)